mkdocs.yml

```diff
@@ -118,7 +121,8 @@ markdown_extensions:
   - pymdownx.superfences
   - pymdownx.tabbed:
       alternate_style: true
-      slugify: !!python/object/apply:pymdownx.slugs.slugify  # NOTE: insiders only
+      slugify:
+        !!python/object/apply:pymdownx.slugs.slugify # NOTE: insiders only
```

Comment:
Double-checking: does this change any functionality?

Reply:
Reverting in case